src/CloseButton.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import IconButton from './IconButton'
+import PropTypes from 'prop-types'
+
+const CloseButton = (props) => (
+  <IconButton {...props} name='closeLight' title={props.title} onClick={props.onClick} size={props.size} />
+)
```

Comment:
Nice! Again here, setting `title`, `onClick`, and `size` are redundant since you set `{...props}`